models/schema_site_bd.go

```diff
@@ -6,11 +6,12 @@ type SiteBd struct {
 	Value map[string]interface{} `json:",omitempty"`
 }
 
-func NewSchemaSiteBd(ops, path string, anpRef map[string]interface{}, host bool) *SiteBd {
+func NewSchemaSiteBd(ops, path, mac string, anpRef map[string]interface{}, host bool) *SiteBd {
 	var externalepgMap map[string]interface{}
 	externalepgMap = map[string]interface{}{
 		"bdRef":            anpRef,
 		"hostBasedRouting": host,
+		"mac":              mac,
```

Comment:
What happens to when payload sent is set to empty string? Did you verify no validation errors are returned?